docs/static/monitoring/monitoring-output.asciidoc

```diff
@@ -28,6 +28,8 @@ the following default configuration is used:
 monitoring.elasticsearch.hosts: [ "http://localhost:9200" ]
 ---------------------------------------------------
 
+//TODO:  Verify the following sentence:
```

Comment:
[Line 16](https://github.com/elastic/logstash/pull/11789/files#diff-6c9faa613d64a8ecf6193e94509afafbR16) Do we want to surface this further up? This content was dropped down in the restructure, and it feels like it might be something we want to surface higher.